test/embind/embind.test.js

```diff
@@ -1294,6 +1294,20 @@ module({
             value = cm.embind_test_optional_small_class_arg(undefined);
             assert.equal(-1, value);
         });
+
+        test("std::optional args can be omitted", function() {
```

Comment:
How about arguments that have default values?

Reply:
Embind doesn't really support default C++ arguments. Functions with default arguments just look like regular functions to embind. e.g. `foo(int a = 100)` is just `foo(int a)` in embind's view of things.